migrations/3_deploy_contracts.js

```diff
@@ -20,13 +19,7 @@ module.exports = function(deployer, network) {
         const controller = await lpDeployer.deployController()
         const token = await lpDeployer.deployAndRegister(LivepeerToken, "LivepeerToken")
         await lpDeployer.deployAndRegister(Minter, "Minter", controller.address, config.minter.inflation, config.minter.inflationChange, config.minter.targetBondingRate)
-
-        if (network === "development" || network === "testrpc" || network === "parityDev" || network === "gethDev") {
-            await lpDeployer.deployAndRegister(IdentityVerifier, "Verifier", controller.address)
-        } else {
-            await lpDeployer.deployAndRegister(LivepeerVerifier, "Verifier", controller.address, config.verifier.solvers, config.verifier.verificationCodeHash)
-        }
-
+        await lpDeployer.deployAndRegister(LivepeerVerifier, "Verifier", controller.address, config.verifier.solvers, config.verifier.verificationCodeHash)
```

Comment:
I don't think there is any practical use for `IdentityVerifier` anymore because even in tests we can just use the `LivepeerVerifier` and if we want to skip verification we would just not have a solver callback to the contract with a result.